src/main/java/cceclipseplugin/core/CCIgnore.java

```diff
@@ -141,7 +151,7 @@ private void serverCleanUp(IProject p) {
 			for (String ignored : ignoredFiles) {
 				if (path.startsWith(ignored) || path.equals(ignored)) {
 					// send delete request for fileID
-					System.out.println(String.format("Cleaning up %s from server", path));
+					Logger.getInstance().log(IStatus.INFO, String.format("Cleaning up %s from server", path));
```

Comment:
Yeah, this would probably be better. My initial thought was to make it similar to System.out so that we could avoid having to create a field for the logger in each class, but I think it would look cleaner